staging/src/kubevirt.io/api/snapshot/v1alpha1/types.go

```diff
@@ -236,6 +236,9 @@ type VolumeBackup struct {
 
 	// +optional
 	VolumeSnapshotName *string `json:"volumeSnapshotName,omitempty"`
+
+	// +optional
+	NoRestore *bool `json:"noRestore,omitempty"`
```

Comment:
Not sure about this field.  I don't think restore logic should be encoded in the snapshot API.  Why not look up the volume by name in the vm spec that is part of the VMSnapshot to get that it's a MemoryDump PVC and ignore or not based on whatever we feel is best to do at the time?